### DISCUSSION TITLE
Authors list on CMT

### DISCUSSION THREAD

**Discussion by dskhanirfan** [FETS-AI/Challenge#104] [General]
The primary author can add all the contributing authors or every author has to register on CMT separately?

I am trying to add contributing authors but I get the following error
Unauthorized (401): Authorization has been denied for this request.

**Comment by ujjwalbaid0408** [comment]
Primary authors can add all the contributing authors. Can you retry adding the other co-authors?

**Reply by dskhanirfan** [reply]
ok.
